docs/wiki/security-process/security-team.md

```diff
@@ -8,11 +8,13 @@
     - Tracking issue: [mimblewimble/grin-pm/#258](https://github.com/mimblewimble/grin-pm/issues/258)
 
 ## Summary
+
 [summary]: #summary
```

Comment:
Are these lines even necessary? AFAIK linking to #summary works without this line.

Reply:
Ah it's the RFC...

Reply:
Yes, it is the RFC